src/components/AmountCard.tsx

```diff
@@ -72,6 +72,7 @@ export function AmountCard(props: {
     initialOpen?: boolean;
     isAmountEditable?: boolean;
     setAmountSats?: (amount: bigint) => void;
+    isSwap?: boolean;
```

Comment:
Should just call this something like "skipWarnings", we have another reason to skip in the future